gfauto/gfauto/cov_from_gcov.py

```diff
@@ -87,12 +94,21 @@ def main() -> None:
     gcov_prefix_dir = os.path.abspath(gcov_prefix_dir)
     gcov_prefix_dir = os.path.normpath(gcov_prefix_dir)
 
+    gcov_tags = (
+        ["functions", "start_line", "execution_count"]
+        if parsed_args.gcov_functions
+        else ["lines", "line_number", "count"]
+    )
+
     data = cov_util.GetLineCountsData(
         gcov_path=gcov_path,
-        gcov_uses_json_output=parsed_args.gcov_uses_json,
+        gcov_uses_json_output=True
+        if parsed_args.gcov_functions
+        else parsed_args.gcov_uses_json,
```

Comment:
I think it would be better to pass parsed_args.gcov_uses_json here, and to check earlier whether this is inconsistent with parsed_args.gcov_functions, quitting with an error if there is an inconsistency.

Reply:
I remove the "if" and added a check with an error message instead.